packages/legend-art/style/base/_panel.scss

```diff
@@ -77,6 +77,18 @@
     @include ellipsisTextOverflow;
   }
 
+  &__header__badge__label {
```

Comment:
is the badge here pertained to only the context of panel? or is this generic badge which we could separate from panel?